samples/testing.fs

```diff
@@ -0,0 +1,45 @@
+module samples.unittest
+
+open Microsoft.Azure.WebJobs
+open DurableFunctions.FSharp
+open System
+
+[<FunctionName("SayHelloTest")>]
+let SayHello([<ActivityTrigger>] name) = 
+    sprintf "Hello %s!" name
+
+[<FunctionName("HelloSequenceTest")>]
+let RunWorkflow ([<OrchestrationTrigger>] context: DurableOrchestrationContextBase) = 
```

Comment:
What would you think of the following style of the test:
- Define an activity (as in `Typed` example)
- Define an orchestrator value. Don't use the activity directly but accept it as a parameter.
- Workflow Function calls the orchestrator with the real activity as the argument
- The test calls the orchestrator with a dummy activity as the argument, which returns hard-coded values. No mocks needed.
Could you try this out and let me know your opinion?

Reply:
Yep - makes sense - I'll try that and see how it goes